### ISSUE TITLE
Error: mismatched types with Rust nightly

### ISSUE THREAD

**Issue by joerg-krause** [plietar/rust-protobuf-macros#6]
Running Rust nightly:

``` sh
$ rustc --version
rustc 1.7.0-dev (a79008f4f 2016-01-07)
$ cargo build
```

Output:

``` sh
<std macros>:3:1: 3:39 error: mismatched types:
 expected `()`,
    found `core::result::Result<_, _>`
(expected (),
    found enum `core::result::Result`) [E0308]
<std macros>:3 $ crate:: result:: Result:: Ok ( val ) => val , $ crate:: result:: Result::
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/joerg/.cargo/git/checkouts/rust-protobuf-macros-0155ebac4efea846/master/src/util.rs:157:17: 157:42 note: in this expansion of try! (defined in <std macros>)
```

I don't know much about Rust (yet), so I cannot help here.

**Comment by plietar** [comment]
Fixed by ade4282 
Thanks for reporting.

**Comment by joerg-krause** [comment]
Many thanks!
